iree/base/logging.cc

```diff
@@ -30,6 +34,16 @@ namespace internal {
 
 namespace {
 
+#ifdef __ANDROID__
+// Define equivalent android log levels to map to IREE.
+static const int kStatusToAndroidLevel[4] = {
```

Comment:
Since this is only used in one place below, you could move it there (like how `kLevelColors` is next to the code that uses it and is a `constexpr` instead of `static const`). That would have the added benefit of not needing a separate `#ifdef __ANDROID__` block